src/Views/Admin/AddTeamComponents/ManageMembers.jsx

```diff
@@ -159,7 +165,9 @@ const ManageMembers = () => {
                 type="text"
                 id="ign"
                 name="ign"
-                onChange={(e) => setIGN(e.target.value)}
+                onChange={(e) =>
+                  setMemberData({ ...memberData, IGN: e.target.value })
+                }
```

Comment:
Supposedly this part kay:

onChange = {(e)=>onChangeDataHandler(e)}

ra unta para mas cleaner.